app/scripts/modules/core/src/application/application.less

```diff
@@ -1,5 +1,11 @@
 @import (reference) "~core/presentation/less/imports/commonImports.less";
 
+application {
```

Comment:
A class would work, but we'd have to put it on the component itself, i.e. `<application class="application"/>`. I bet we'll remember (because nothing will be scrollable on the page).